docs/docs/cloud/cloud-cli.mdx

```diff
@@ -260,7 +375,16 @@ meltano-cloud job stop --execution-id 15e1cbbde6b2424f86c04b237291d652
 
 :::info
 
-  <p>At the moment, executions stopped manually will be marked as <i>Failed</i> in the output of <a href="#history"><code>meltano-cloud history</code></a>. See <a href="https://github.com/meltano/meltano/issues/7697">#7697</a> for more details.</p>
+{" "}
```

Comment:
Is this needed here?

Reply:
I don't believe so, but I'm not super familiar with MDX or docusaurus. I removed it and it doesn't appear to have made a difference in the deploy preview, so I think we're good to leave it out.